core/src/main/java/fj/P2.java

```diff
@@ -342,6 +344,34 @@ public B _2() {
     return (a, b) -> f.f(P.p(a, b));
   }
 
+  /**
+   * Polyomorphic lens targeted on _1.
+   */
+  public static <A, B, C> PLens<P2<A, B>, P2<C, B>, A, C> _1pLens() {
+    return pLens(__1(), a -> p2 -> P.p(a, p2._2()));
+  }
+
```

Comment:
for reference, the Monocle library use ```first``` and ```second``` for the lenses on scala tuple:
https://github.com/julien-truffaut/Monocle/blob/master/core/src/main/scala/monocle/std/Tuple2.scala